conf.py

```diff
@@ -128,7 +129,11 @@
 github_user = 'odoo'
 github_project = 'documentation'
 
-locale_dirs = ['locale/']
+locale_dirs = ['../locale/']
+sphinx.transforms.i18n.docname_to_domain = (
+    sphinx.util.i18n.docname_to_domain
+) = lambda docname, compact: docname.split('/')[1 if docname.startswith('applications/') else 0]
```

Comment:
Maybe add a comment to explain the unholy mess we're wrecking here (e.g. what's the issue, how `docname_to_domain` is involved and what this monkeypatch achieve).

Reply:
Or at least expound on it a bit more in the commit message since you have a commit dedicated to the issue.